harness/features/track.local.test.ts

```diff
@@ -0,0 +1,378 @@
+import { getConnectionStringForProxy, forEachSDK, describeIf, createClient, createUser } from '../helpers'
+import { Capabilities, SDKCapabilities, latestNodeJsSdkVersion } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { config } from '../mockData/index'
+import nock from 'nock'
+
+jest.setTimeout(1000000)
+
+const scope = getServerScope()
+
+describe('Track Tests - Local', () => {
+    const mockEvents = jest.fn()
+    const validUserId = 'user1'
+    forEachSDK((name) => {
+        let url: string
+        const capabilities: string[] = SDKCapabilities[name]
+        const clientId: string = uuidv4()
+        const mockServerUrl = `http://host.docker.internal:${global.__MOCK_SERVER_PORT__}`
```

Comment:
this should be `http://${process.env.DOCKER_HOST_IP ?? 'host.docker.internal'}:${global.__MOCK_SERVER_PORT__}` :P

Reply:
ofcourse